loops-and-arrays.js

```diff
@@ -42,7 +42,7 @@ for (i = 0; i < numbers.length; i++) {
 
 const squares = [30, 2, 8, 24, 11];
 for (i = 0; i < squares.length; i++) {
-    console.log(squares[i] * squares[i] * squares[i]);
+    console.log(squares[i] ** 3);
 }
 
```

Comment:
Wederom hetzelfde, laats je console.log buiten de loop.
Ik begrijp alleen niet zo goed waarom je de console.log hier 2x aanspreekt. Beide geven dezelfde uitkomst en ik zou lekker voor de laatste gaan, die is het makkelijkst. Maar console.log buiten je loop

`const square = [30, 2, 8, 24, 11];
for (i = 0; i < square.length; i++) {
    square[i] = square[i] * square[i] * square[i];

}
console.log(square);`